### ISSUE TITLE
Local dev on Linux

### ISSUE THREAD

**Issue by MrSunshyne** [saasify-sh/puppeteer-serverless#1]
hey @chrisvxd, i'm currently trying this and it doesn't seem to work locally on Linux. Any tips? 

Thanks  !

**Comment by chrisvxd** [comment]
Right, that's because it was never tested/built for linux dev environments (short sighted I know).

If you look at [this file](https://github.com/saasify-sh/puppeteer-serverless/blob/master/src/options.ts), you'll see it uses the locally installed version of Chrome. If it's not Windows, it assumes it's MacOS.

Where is your local copy of Chrome installed? You can override the behaviour

```js
import puppeteer from 'puppeteer-serverless'

const isDev =  process.env.NODE_ENV === 'development';
const browser = isDev ?
  await puppeteer.launch({ executablePath: 'path/to/chrome' }) :
  await puppeteer.launch();
```

But if you let me know, I can add it to the codebase

**Comment by MrSunshyne** [comment]
@chrisvxd the location could be different on different linux distos, 
but it's usually in `/usr/bin/google-chrome`

I tried it the following way
```js
import puppeteer from 'puppeteer-serverless';
import renderSocialImage from 'puppeteer-social-image';

const gen = async() => {
  const isDev =  process.env.NODE_ENV === 'development';
  const browser = isDev ?
    await puppeteer.launch({ executablePath: '/usr/bin/google-chrome' }) :
    await puppeteer.launch();

  await renderSocialImage({
    template: "article",
    templateParams: {
      unsplashId: "WV4B_aVj0aQ",
      title: "My awesome title",
      subtitle: "Is Maths related to science?",
      eyebrow: "brows",    
    },
    output: "output.png",
    browser
  });
}

gen()
```

executed using 
```json
"scripts": {
        "start": "NODE_ENV=development babel-node main.js",
  },
```

```
npm run start
```

An output.png file is indeed generated, but it's a transparent png with no content. I'm not sure where to look from here...

**Comment by chrisvxd** [comment]
Hmm this has me a bit stumped without running a linux VM, honestly. Do you get the same output for all the examples?

Also, a word of advice for lambda - if you place your `browser` variable at the root of the file as a `let`, and only assign it if it's undefined in your function, lambda will reuse the browser instance between invocations.

**Comment by MrSunshyne** [comment]
> without running a linux VM, honestly.
Yes that's understandable.

All the examples work great when not using the puppeteer-serverless package, but that's kinda what i'm hoping to achieve. 

I'm trying to deploy it on Netlify Fuctions, using the snippet you provided in the docs. 

But it's resulting in a failed deploy. Logs are here:
https://app.netlify.com/sites/ogimage/deploys/5e9df8cd54311f0006a7f10d

Perhaps this could help shed some light?

**Comment by chrisvxd** [comment]
@MrSunshyne not hugely I'm afraid. Did you have any more luck?

I looked at your code and noticed you switched away from this lib

**Comment by MrSunshyne** [comment]
thanks for taking a look @chrisvxd 

I've been trying all sorts of different combination in the hope of getting it to work as a serverless function, but still not getting anywhere. I've gone back to the most basic example that you provided in the docs. See code [here](https://github.com/MrSunshyne/social-image-server/blob/master/functions/social-image.js)

I'm getting the following error message when trying to deploy: 
```
puppeteer-social-image@0.6.0 requires a peer of puppeteer@2.x but none is installed. You must install peer dependencies yourself.
Error: Could not find "puppeteer" module in file: /opt/build/repo/functions/social-image.js.
Please ensure "puppeteer" is installed in the project.
```
Full deploy log [here](https://app.netlify.com/sites/ogimage/deploys/5ea015a044f3ee0006754bda)

I was hoping to get at least the docs example working before I can proceed to adding stuff :(
